Apps/Sandcastle/gallery/3D Tiles Point Cloud Styling.html

```diff
@@ -109,8 +109,13 @@
     color : "color() * abs(${temperature} - 0.5)"
 });
 
-addStyle('Cos', {
-    color : "color() * abs(cos(${temperature} + TILES3D_TILESET_TIME))"
+addStyle('Trigonometric Functions', {
+    color : "color() * radians(cos(${temperature})) + color() * sin(${temperature}) + color() * tan(${temperature}) + 30"
+
+});
+
+addStyle('Arc Trigonometric Functions', {
+    color : "color() * degrees(acos(${temperature})) + color() * asin(${temperature}) + color() * atan(${temperature}) - 30"
```

Comment:
These two samples still produce pure white. Try to tweak them a bit.

For the first one removing the +30 helps. For the second one both the -30 and the degrees are keeping it white. When the channels of the color are greater than 1.0 it will be white.